modules/tinymce/CHANGELOG.md

```diff
@@ -41,6 +41,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Formatter API `canApply` was not returning `false` when the selection was in a noneditable context. #TINY-9678
 - When dragging image elements and dropping the image in the editor the `dragend` event would sometimes not fire on firefox. #TINY-9694
 - It was possible to remove links in noneditable contents with the 'unlink' editor command. #TINY-9739
+- Inserting table consecutively without focus in the editor would result in the table being inserted at the wrong position. #TINY-3909
```

Comment:
There is no test that covers this specific case. So setup a editor without focus insert a table and assert.

Reply:
test created